src/send/selectors.ts

```diff
@@ -16,11 +16,21 @@ export const isSendingSelector = (state: RootState) => {
 
 export const inviteRewardCusdSelector = (state: RootState) => state.send.inviteRewardCusd
 
-export const inviteRewardsEnabledSelector = (state: RootState) => state.send.inviteRewardsEnabled
+export const inviteRewardsTypeSelector = (state: RootState) => {
+  switch (state.send.inviteRewardsVersion) {
+    case 'v4':
+      return 'nft'
+    case 'v5':
+      return 'cUSD'
+    default:
+      return 'none'
```

Comment:
oh interesting selector, usually we've had the same values on remote config as in the code but i think this makes sense too since the version will make more sense to the product team. can we use an enum instead though, so that we can easily see all the available values? then when we use the type we can reference the enum instead of the actual value

Reply:
aside: i'd also recommend adding a description of this value in the remote config console, to explain the acceptable values (e.g. when Ellen goes to update this value, she needs to know what the available options are)